rust/xaynet-server/src/services/messages/mod.rs

```diff
@@ -55,6 +58,17 @@ impl PetMessageHandler {
         self.message_parser.call(data).await
     }
 
+    async fn handle_multipart(
+        &mut self,
+        message: Message,
+    ) -> Result<Option<Message>, ServiceError> {
+        poll_fn(|cx| {
```

Comment:
Is it possible to call this function as with the other functions?

like:
```rust
poll_fn(|cx| self.multipart_handler.poll_ready(cx)).await?;
```

Reply:
Oh actually yes, thank you!